src/status_im2/contexts/activity_center/notification/contact_requests/view.cljs

```diff
@@ -0,0 +1,113 @@
+(ns status-im2.contexts.activity-center.notification.contact-requests.view
+  (:require [quo2.core :as quo]
+            [react-native.core :as rn]
+            [status-im2.constants :as constants]
+            [status-im2.contexts.activity-center.notification.common.view :as common]
+            [status-im2.contexts.activity-center.notification.contact-requests.events]
```

Comment:
We shouldn't require event namespaces in views. Events should be registered once in the global re-frame registry.

The require should go in `status-im.events`.